pages/vpc/reference-content/understanding-nacls.mdx

```diff
@@ -0,0 +1,109 @@
+---
+meta:
+  title: Understanding Network ACLs
+  description: Learn how to Network Access Control Lists (NACL) filter inbound and outbound traffic between the different Private Networks of your VPC. Understand concepts, best practices, and key use cases.
+content:
+  h1: Understanding Network ACLs
+  paragraph: Learn how to Network Access Control Lists (NACL) filter inbound and outbound traffic between the different Private Networks of your VPC. Understand concepts, best practices, and key use cases.
+tags: vpc nacl network-access-control-list default-rule stateless inbound outbound port
+dates:
+  validation: 2025-03-26
+  posted: 2025-03-26
+categories:
+  - network
+---
+
+<Message type="important">
+NACLs are currently in Public Beta, and configurable only via the [VPC API](https://www.scaleway.com/en/developers/api/vpc/) and developer tools. This feature will be coming soon to the Scaleway console.
+</Message>
+
+Every VPC has a Network **A**ccess **C**ontrol **L**ist (NACL). This list is composed of stateless rules to control the flow of traffic between the Private Networks of the VPC. By default, at first, the list contains no rules, and therefore traffic is allowed to flow unrestrictedly. You can add rules to restrict traffic flow according to your requirements.
+
+This document sets out general information and best practices about Scaleway VPC NACLs.
+
+## Network ACL overview
+
+Every Scaleway VPC has a Network ACL. In its initial state, it contains no rules. Thanks to [managed routing](/vpc/reference-content/understanding-routing/), which automatically routes traffic between Private Networks, traffic is freely routed between the Private Networks of the VPC.
+
+When you start adding rules to your NACL, traffic flow is restricted between certain sources and destinations within the VPC, according to the rules you set. A default rule is added to the NACL, which dictates the action to take on traffic that does not match any rules in the list: it can either be **allowed** or **denied**.
+
+<Lightbox src="scaleway-nacl-diag-simple.webp" alt="A schema shows how the NACL sits at the intersection of two Private Networks in a Scaleway VPC" />
+
+NACL rules are stateless, meaning that the state of connections is not tracked, and return traffic is not automatically allowed, just because the outbound request was allowed. Explicit rules are required for each direction of traffic.
+
+NACLs only control traffic as it enters or exits the Private Network(s) of a VPC. They do not:
+- Filter traffic between resources attached to the same Private Network
+- Filter traffic from/to the public internet (for this, use [security groups](/instances/how-to/use-security-groups/) for Instances, or equivalent features for [other resource types](/ipam/reference-content/)).
+
+The diagram below shows how a NACL allows an Instance on Private Network A to send a packet to an Instance on Private Network.
```

Comment:
My next suggestions about replacing "an NACL" with "a NACL" are based on this sentence.